client/my-sites/upgrades/domain-management/list/item.jsx

```diff
@@ -24,7 +25,7 @@ const ListItem = React.createClass( {
 
 					<div className="domain-management-list-item__meta">
 						<span className="domain-management-list-item__type">{ this.getDomainTypeText() }</span>
-
+						{ this.props.domain.type !== 'WPCOM' ? this.isDomainExpired( this.props.domain ) : null }
```

Comment:
`isDomainExpired` gives a sense that it will return a boolean, however it returns a component, maybe we should rename it.

Also, no need to use `if ? fn : null`, we can just do `{ isRegisteredDomain( this.props.domain ) && this.domainExpiration() }`